File: implementation/scipamato/core/sync/src/test/java/ch/difty/scipamato/core/sync/jobs/newsletter/NewsletterItemWriterTest.java
package ch.difty.scipamato.core.sync.jobs.newsletter;

import org.jooq.DSLContext;

import ch.difty.scipamato.core.sync.jobs.AbstractItemWriterTest;

class NewsletterItemWriterTest extends AbstractItemWriterTest<PublicNewsletter, NewsletterItemWriter> {

    @Override
    protected NewsletterItemWriter newWriter(DSLContext dslContextMock) {
        return new NewsletterItemWriter(dslContextMock);
    }
}

File: implementation/scipamato/core/sync/src/test/kotlin/ch/difty/scipamato/core/sync/jobs/newsletter/NewsletterItemWriterTest.kt
package ch.difty.scipamato.core.sync.jobs.newsletter

import ch.difty.scipamato.core.sync.jobs.AbstractItemWriterTest
import org.jooq.DSLContext

internal class NewsletterItemWriterTest : AbstractItemWriterTest<PublicNewsletter, NewsletterItemWriter>() {
    override fun newWriter(dslContextMock: DSLContext) = NewsletterItemWriter(dslContextMock)
}
